### ISSUE TITLE
why detach history feat？

### ISSUE THREAD

**Issue by shaoqb** [Divadi/SOLOFusion#22]
https://github.com/Divadi/SOLOFusion/blob/683edce81b619098d1ba143d7b15b1e6aa23337a/mmdet3d/models/detectors/solofusion.py#L377

**Comment by Divadi** [comment]
This is to prevent backpropagation through time, which would incurr significant computational costs (and may not even work after backpropagation is done in the previous training step).
